MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+* In the decade view, when the (2022) year cell is selected then the selected date value will be 01-01-2022.
+* In the century view, when the (2020-2029) decade cell is selected then the selected date value will be 01-01-2020.
+
+
+## Multiple selection
+The `Multiple` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Multiple`. In this Selection, you can able to select a multiple dates from the cell. If you want to remove the selected cell by clicking the same cell again.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Multiple">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Multiple;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Multiple selection in .NET MAUI Calendar.](images/selection/maui-multiple-selection.png)
+
+
+## Range selection
+The `Range` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Range`. In this Selection, you can able to select a range of dates from the cell by interacting the cell either tap or swipe action. Swipe action can only performed by setting the [enableSwipeSelection] property to `true` in the calendar.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Range">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Range;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Range selection in .NET MAUI Calendar.](images/selection/maui-range-selection.png)
+
+>**NOTE**
+* In the Year, Decade and Century views can able to select the cells, only when the `allowViewNavigation` property is set to false.
+* In this scenario, the `selection changed` callback will return the first  and last date of the month, year, decade or century of the selected cell when the seletionmodeis set to `range `.
+Eg: 
+* In the year view, when the range is selected as Sep - Dec, then the range value will be 01-09-2022 to 31-12-2022.
+* In the decade view, when the range is selected as 2022 - 2025, then the range value will be 01-01-2022 to 31-12-2025.
+* In the century view, when the range is selected as 2020-2029 to 2030-2039, then the range value will be 01-01-2020 to 31-12-2039.
+
+### Range Selection Direction
+The Range Selection Direction is specified in the calendar property enumeration which is uesd to select the range based on the direction. The `RangeSelectionDirection` provides five types of direction such as `Default`, `Forward`, `Backward`, `Both` and `None`. The default `RangeSelectionDirection` is `Default` in the SfCalendar.
```

Comment:
The Range Selection Direction is specified in the calendar property enumeration, which is used to select the range based on the direction. The `RangeSelectionDirection` provides five types of direction such as `Default,` `Forward,` `Backward,` `Both,` and `None.` The default `RangeSelectionDirection` is `Default` in the SfCalendar.

Reply:
Content modified